docs/CustomResources.md

```diff
@@ -244,16 +246,42 @@ the `IndexerCluster` resource provides the following `Spec` configuration parame
 | replicas   | integer | The number of indexer cluster members (defaults to 1) |
 
 
+## MonitoringConsole Resource Spec Parameters
+
+```
+apiVersion: enterprise.splunk.com/v1
```

Comment:
Sorry for not being clearer

* Firstly, this PR should be targeted towards feature branch `feature-MC/CRD `and not `develop`. This is because all relevant MC development is happening on the feature branch. Please retarget the PR to feature branch
* The examples in the PR are currently inconsistent. For instance, example shown under `docs/Examples.md` has api version set as `enterprise.splunk.com/v2`, where as here we have it set as v1. 
* The latest develop has v2 (due to changes introduced by App framework), and hence this PR, at this current stage should also point to v2, rather than v1. Kriti will be updating the feature branch with latest develop to reflect this
* And my v3 comment was meant to be informational. Kriti will eventually bump up the apiVersion to v3 at a later stage, to reflect her new changes to the CRs. At this point no action to be taken wrt v3 in this PR